typescript/cli/src/deploy/core.ts

```diff
@@ -87,12 +87,12 @@ export async function runCoreDeploy({
   artifactsPath?: string;
   outPath: string;
   skipConfirmation: boolean;
-  dryRun: boolean;
+  dryRun: string;
 }) {
   const context = dryRun
     ? await getDryRunContext({
         chainConfigPath,
-        chains,
+        chains: [dryRun],
```

Comment:
should `dryRun` be `string | undefined`? tbf this is part of a larger yargs stronger typing effort cc @nbayindirli

Reply:
Does this imply `dryRun` flag is an optional list of strings now?

Reply:
its a single chain